src/entities/comment/api/comments.ts

```diff
@@ -0,0 +1,27 @@
+import { fetcher } from "@/base/api"
+import type * as CommentModels from "@/entities/comment/model"
+
+export async function fetchCommentsByPostId({ postId }: CommentModels.FetchCommentsByPostId.Payload) {
+  const response = await fetcher.get(`/comments/post/${postId}`)
+  return response.json<CommentModels.FetchCommentsByPostId.Response>()
+}
```

Comment:
json 으로 한번 뽑는 부분을 fetcher.get 안으로 넣어서 아래처럼 쓸 수 있으면 더 간편했을 것 같아요!
```suggestion
export async function fetchCommentsByPostId({ postId }: CommentModels.FetchCommentsByPostId.Payload) {
  return fetcher.get<CommentModels.FetchCommentsByPostId.Response>(`/comments/post/${postId}`);
}
```